util/missingcrypto.txt

```diff
@@ -572,6 +572,41 @@ ENGINE_setup_bsd_cryptodev(3)
 ENGINE_unregister_EC(3)
 ENGINE_unregister_pkey_asn1_meths(3)
 ENGINE_unregister_pkey_meths(3)
+EVP_ASYM_CIPHER-RSA(7)
+EVP_CIPHER-AES(7)
+EVP_CIPHER-ARIA(7)
+EVP_CIPHER-BF(7)
+EVP_CIPHER-DES(7)
+EVP_CIPHER-CAMELLIA(7)
+EVP_CIPHER-CAST5(7)
+EVP_CIPHER-ChaCha20(7)
+EVP_CIPHER-ChaCha20-Poly1305(7)
+EVP_CIPHER-DES(7)
+EVP_CIPHER-IDEA(7)
+EVP_CIPHER-RC2(7)
+EVP_CIPHER-RC4(7)
+EVP_CIPHER-RC5(7)
+EVP_CIPHER-SEED(7)
+EVP_CIPHER-SM4(7)
+EVP_KDF-KBKDF(7)
+EVP_KDF-SSKDF(7)
+EVP_KDF-TLS1-PRF(7)
+EVP_KEYEXCH-DH(7)
+EVP_KEYEXCH-ECDH(7)
+EVP_KEYEXCH-DSA(7)
+EVP_KEYEXCH-RSA(7)
+EVP_KEYEXCH-X25519(7)
+EVP_KEYEXCH-X448(7)
+EVP_KEYMGMT-DH(7)
+EVP_KEYMGMT-EC(7)
+EVP_KEYMGMT-DSA(7)
+EVP_KEYMGMT-RSA(7)
+EVP_KEYMGMT-X25519(7)
+EVP_KEYMGMT-X448(7)
+EVP_MD-BLAKE2(7)
+EVP_MD-SM3(7)
+EVP_SIGNATURE-DSA(7)
+EVP_SIGNATURE-RSA(7)
```

Comment:
Why are these appearing in missingcrypto at all????

Reply:
Because that's where find-doc-nits looks? We could have a different file for man7 stuff if you prefer, but I'd make that a separate PR. Feel free to... :wink:

Reply:
Your assumptions are quite wrong.  find-doc-nits finds *links to missing man-pages*, which is why every line is a man-page reference.  It *also* checks public function names specifically against man-pages in the man3 section, *but is not limited to that*.

So you have to mainly think of it in terms of man-pages, and *then* that man3 pages must match our public functions.

Reply:
Hmmm. Ok. In which case - why are you listing 2 EVP_MD pages above? Shouldn't those pages be within the scope of this PR?

Reply:
I've added what was missing, and added the legacy stuff while I was at it

Reply:
Regarding missingcrypto.txt, you're right that it was originally meant to collect only public functions, that's also how it's used in some parts of find-doc-nits (the printem call).  So really, man7 page references should be collected in a separate file, and processed separately.

That's a topic for yet another PR